src/app/config/config.default.ts

```diff
@@ -126,5 +126,13 @@ export const defaultConfig: Config = {
     previewProgressCircle: false,
     screenSleepCommand: 'xset dpms force standby',
     screenWakeupCommand: 'xset s off && xset -dpms && xset s noblank',
+    window: {
+      width: null,
+      height: null,
+      x: null,
+      y: null,
```

Comment:
numbers would make more sense here i guess.

Reply:
defaults should be: `800, 480, 0, 0`, since that is the main resolution OctoDash is being developed for.